nemo/collections/tts/modules/submodules.py

```diff
@@ -709,18 +709,29 @@ class SpeakerEncoder represents speakers representation.
     This module can combine GST (global style token) based speaker embeddings and lookup table speaker embeddings.
     """
 
-    def __init__(self, lookup_module=None, gst_module=None):
+    def __init__(self, lookup_module=None, gst_module=None, precompute=False, precompute_embedding_dim=None):
         """
         lookup_module: Torch module to get lookup based speaker embedding
         gst_module: Neural module to get GST based speaker embedding
+        precompute: Use precompute speaker embedding
         """
         super(SpeakerEncoder, self).__init__()
+
+        # Multi-speaker embedding
         self.lookup_module = lookup_module
+
+        # Reference speaker embedding
         self.gst_module = gst_module
 
+        if precompute:
+            self.precomputed_emb = torch.nn.Parameter(torch.empty(precompute_embedding_dim))
```

Comment:
Would be nice to have a method in the class that shows how to overwrite this value.

Reply:
Add function`overwrite_precomputed_emb()`